packages/survey/src/Question/Item/Item.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box, Icon, Image, Text, useCheckbox, useRadio } from '@chakra-ui/react'
+import { FaCheck } from 'react-icons/fa'
+import { IoEllipse, IoRadioButtonOff, IoSquareOutline } from 'react-icons/io5'
```

Comment:
Can't we use `IoCheckmark` here?

Reply:
we could but this is the original one, 
![image](https://user-images.githubusercontent.com/42379840/122874433-476c3a00-d333-11eb-882b-e85ca6c576aa.png)
its bolder than the IoCheckmark, if will be accepted from design its okay ;)

Reply:
I am absolutely fine with it!
I just wanted to say that we should try using only Ionicons. But in cases like this, I can follow you to use the thicker style of FontAwesome. Especially since this checkmark is a separated icon.